plugins/ccgx/fu-plugin-ccgx.c

```diff
@@ -26,6 +26,7 @@ fu_plugin_ccgx_init(FuPlugin *plugin)
 	fu_context_add_quirk_key(ctx, "CcgxFlashRowSize");
 	fu_context_add_quirk_key(ctx, "CcgxFlashSize");
 	fu_context_add_quirk_key(ctx, "CcgxImageKind");
+	fu_context_add_quirk_key(ctx, "CcgxDmcTrigCode");
```

Comment:
I think this should probably be `CcgxDmcTriggerCode` and also documented in README.md

Reply:
Please check [53c653c](https://github.com/fwupd/fwupd/pull/4435/commits/53c653ca9db90a142b95992e167f06f819396a0f)